.storybook/preview.ts

```diff
@@ -6,8 +6,19 @@ import "./theme.css";
 
 const preview: Preview = {
   parameters: {
+    layout: "centered",
     backgrounds: {
-      disable: true,
+      default: "theme",
+      values: [
+        {
+          name: "theme",
+          value: "var(--ink-storybook-theme-color)",
+        },
+        {
+          name: "container",
+          value: "var(--ink-background-container)",
+        },
```

Comment:
`theme` will follow whether you have dark/light Ink theme.

`container` will show you the container color